azext_edge/tests/edge/init/custom_template.json

```diff
@@ -0,0 +1,1050 @@
+{
```

Comment:
is there a reason for saving a template rather than dumping one via az iot ops init and using that? (and deleting it at the end, you can move my delete all file created fixture wherever you need it)

Reply:
One use is a convenient non-standard template for smoke test/user test. But doesn't have to be.

Reply:
Going to generate.